### DISCUSSION TITLE
Just wanna say - I'm really impressed with and appreciate this project

### DISCUSSION THREAD

**Discussion by riotrah** [U-C-S/Hurl#115] [General]
I myself tried to implement something like this with electron by refactoring Browserosaurus to work on Windows, but got stuck trying to get Windows to recognize it as a browser. There _were_ other Windows browser select apps, but coming from Mac, where both Browserosaurus and Velja - and really most apps I've come across - look so good, it was hard to go back to the 90s-looking, and often unmaintained, ones!

What I'm saying is, I appreciate your making this work, iterating on it greatly, engaging with the community and making it so pretty. Plus, keeping that list of alternatives - I've tried them all but have kept using yours daily.

You've done a great job with this - I'm especially impressed knowing this is one of your first software development and open source projects. Look forward to your future works!

**Comment by U-C-S** [comment]
Pleased to hear this 😊

Also, I tried veija a while ago. like how performant it is and couple of features like hold `Fn` to open in fav browser. And its simplicity. I feel Hurl is growing in complexity hence I am doing a rewrite and take some time with that, especially with code.

Rest all, haha I was anxious all the time whenever I writing something out to community, even rn. I feel a bit assured now.

---

offtopic: I encountered couple of your PRs for Calva few weeks ago, when I want to do one for `clojure-lsp`. The World is a small place :D
